Cargo.toml

```diff
@@ -17,7 +17,6 @@ leptos_axum = { version = "0.8" }
 anyhow = "1.0"
 artem = { version = "3.0", default-features = false }
 axum = { version = "0.8", features = ["macros"] }
-base64 = "0.22"
```

Comment:
Thanks for catching that the base64 crate is imported but not currently used lol.